hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/multisync/TestMultipleMetaSync.java

```diff
@@ -49,7 +49,7 @@ void testMultipleMetaStore() throws Exception {
     MockSyncTool2.syncSuccess = false;
     // Initial bulk insert to ingest to first hudi table
     HoodieDeltaStreamer.Config cfg = getConfig(tableBasePath, getSyncNames("MockSyncTool1", "MockSyncTool2"));
-    new HoodieDeltaStreamer(cfg, jsc, fs, hiveServer.getHiveConf()).sync();
+    syncOnce(new HoodieDeltaStreamer(cfg, jsc, fs, hiveServer.getHiveConf()));
```

Comment:
Lots of changes in this PR are using utils or other means to ensure the streamer is shutdown